https support.

alexlod opened the pull request:
I need help with one issue: the security doc I introduce doesn't show up in the top-level menu when I `make html`. Maybe this is a local configuration issue?

Depends on: https://github.com/confluentinc/rest-utils/pull/38

Don't merge until https://github.com/confluentinc/rest-utils/pull/38 is merged.

ewencp commented on docs/security.rst:
```diff
@@ -0,0 +1,5 @@
+
+Security Overview
+-----------------
+The REST proxy currently supports API calls over https. At this time, ZooKeeper security and Kafka security are not yet supported.
```
If you think it makes sense, we could say to check the configuration options (and link to them) for more details.

ewencp commented:
@alexlod Changes look good. To get the security doc in, add it to the table of contents in `index.rst`.

alexlod commented:
Thanks, @ewencp! Addressed your comments (and fixed the issue) in my newest commit. For whatever reason, I didn't need to modify the table of contents in the schema registry to make the top-level menu change -- that threw me off.

ewencp commented:
It wasn't a new file there, but here it was just added.

LGTM once rest-utils patch is in. Anything blocking that one from landing?

ewencp commented:
@alexlod And again, sorry, merged another one of my doc patches and it conflicts.

alexlod commented:
@ewencp no worries. Just pushed a fix.
